qa/L0_model_config/autofill_noplatform_success/onnx/cpu_instance/1/model.onnx

```diff
@@ -0,0 +1,33 @@
+TRTIS:�
+
+INPUT0_INPUT0"Identity
+
+INPUT1_INPUT1"Identity
+
+_INPUT0
+_INPUT1CAST0"Add
+
+_INPUT0
+_INPUT1CAST1"Sub
+!
+CAST0OUTPUT0"Cast*	
+to�
+!
+CAST1OUTPUT1"Cast*	
+to�onnx_int32_int8_int8Z
+INPUT0
+
+var_0
+Z
+INPUT1
+
+var_0
+b
+OUTPUT0
+
+var_1
+b
+OUTPUT1
+
+var_2
+B
```

Comment:
You can use identity model instead for the test, there is identity model that has only 1 input and output, which means we only need to provide one "expected" file (model only has one I/O permutation)

Reply:
Fixed